src/InteractiveWindow/Editor/Commands/InteractiveWindowCommands.cs

```diff
@@ -271,6 +277,13 @@ public void DisplayHelp()
                 _window.Write(HelpIndent);
                 _window.WriteLine(line);
             }
+
+            _window.WriteLine(InteractiveWindowResources.ScriptDirectives);
+            foreach (var line in s_scriptDirectives)
+            {
+                _window.Write(HelpIndent);
+                _window.WriteLine(line);
+            }
```

Comment:
A bug is files for proper implementation of this.
https://github.com/dotnet/roslyn/issues/6441